spring-boot-starter/example-invoice/pom.xml

```diff
@@ -11,6 +11,7 @@
     <spring.boot.version>3.1.1</spring.boot.version>
     <graaljs.version>21.1.0</graaljs.version>
     <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
+    <maven.compiler.release>11</maven.compiler.release>
     <java.version>17</java.version>
```

Comment:
❓This looks weird. Which one is correct? I think latest Spring version needs 17.

Reply:
The correct value is 17 as you correctly questioned. I've updated all spring boot examples to populate property `maven.compiler.release` with 17 version.